Doc/library/unittest.mock.rst

```diff
@@ -1308,9 +1308,23 @@ patch.dict
     :func:`patch.dict` can also be called with arbitrary keyword arguments to set
     values in the dictionary.
 
-    :func:`patch.dict` can be used as a context manager, decorator or class
-    decorator. When used as a class decorator :func:`patch.dict` honours
-    ``patch.TEST_PREFIX`` for choosing which methods to wrap.
+:func:`patch.dict` can be used as a context manager, decorator or class
+decorator. When used as a class decorator :func:`patch.dict` honours
+``patch.TEST_PREFIX`` for choosing which methods to wrap.
+
+    >>> foo = {}
```

Comment:
What do you think about moving this example before the "When used as a class decorator :func:`patch.dict` honours [...]" part? Seeing this example after reading a sentence about using `patch.dict()` as a class decorator seems confusing to me :)

Reply:
Great, it's a good idea. I didn't want break the existing paragraphs. I will change.